src/app/[locale]/(home)/occasions/_components/forms/create-occasion-form.tsx

```
@@ -0,0 +1,146 @@
+    // Form submission
+    const onSubmit = (values: CreateOccasionFormValues) => {
+        // Create form data
+        const formData = new FormData();
+        formData.append("name", values.name);
+        formData.append("image", values.image);
+
+        // Trigger mutation
+        createMutation.mutate(formData, {
+            onSuccess: (response) => {
+                if (response.ok) {
+                    const successPayload = response.payload as CreateOccasionResponse;
```

ESLint

[error] 'successPayload' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)